src/Facades/ColorManager.php

```diff
@@ -31,7 +31,7 @@ public static function getColorAsArray(array $options, string $optionKey): array
             try {
                 $rgbColor = Rgb::fromString($getColor);
             } catch (\Exception) {
-                $rgbColor = Rgb::fromString(Qr::getDefaultOptions()[$optionKey]);
+                $rgbColor = Rgba::fromString(Qr::getDefaultOptions()[$optionKey]);
```

Comment:
why using `Rgba` for none Rgba colors? notice the 

`if (str($getColor)->startsWith('rgba')) {`

some colors dont use rgpa

Reply:
Yes, we need to implement a color format check

Reply:
ya but the condition is way up there line 22 :) it will never reach this part! and it already has its own try catch

Reply:
Sure it reaches this part of code, if you type in some random strings

Reply:
My new commit solves the problem with the distinction between rgb and rgba colors